docs/basic-usage.rst

```diff
@@ -491,6 +491,29 @@ alter the ``contain()`` list for the pagination query:
         return $this->Crud->execute();
     }
 
+Bulk Actions
+------------
+
+There are bulk actions which can be easily used with crud view.
+
+To set up crud action in controller do something like this in initialize method.
+
+.. code-block:: php
+    $this->loadComponent('Crud.Crud', [
+        'actions' => [
+            'deleteAll' => [
+                'className' => 'Crud.Bulk/Delete',
+            ],
+        ]
+    ]);
+
+Then you need to config scaffold to display select with actions.
+
+.. code-block:: php
+    $action->config('scaffold.bulk_actions', [
+        Router::url(['action' => 'deleteAll']) => __('Delete records'),
```

Comment:
@josegonzalez Doesn't using something like 

```php
'deleteAll' => ['title' => 'Delete Record', 'url' => ['action' => 'deleteAll']]
``` 
also work? If so that's what we should suggest using as it's cleaner.